app/src/main/java/com/onelist/dto/Category.kt

```diff
@@ -1,7 +1,15 @@
 package com.onelist.dto
 
+/**
```

Comment:
Kotlin allows documentation to be written like this to explain functionality with having tags to explain who created the code with specific code corresponding to what they represent

Reply:
While KDoc is a good idea, I urge a little caution on this KDoc:

- @author tag is deprecated.  That was useful before we had good version control, back when we couldn't see who made specific changes.  believe it or not, that used to not be a thing...
- Constructor is a given.